administration/src/cards/CardBlueprint.ts

```diff
@@ -125,8 +127,8 @@ export const isValueValid = (cardBlueprint: CardBlueprint, cardConfig: CardConfi
       return isExpirationDateValid(cardBlueprint) || hasInfiniteLifetime(cardBlueprint)
     default: {
       const extensionName = getExtensionNameByCSVHeader(cardConfig, columnHeader)
-      const extension = getExtensions(cardBlueprint).find(({ extension }) => extension.name === extensionName)
-      return extension?.extension.isValid(extension.state) ?? false
+      const extension = cardConfig.extensions.find(extension => extension.name === extensionName)
+      return extension?.isValid(cardBlueprint.extensions) ?? false
```

Comment:
This is the relevant change, we need to check among all extensions configured in the card config, not just the ones actually set.